src/components/filters/filter_menu_item/filter_menu_value_item.ts

```diff
@@ -5,8 +5,8 @@ interface Props {
   value: string;
   isChecked: boolean;
   isSelected: boolean;
-  onMouseEnter: () => void;
   onClick: () => void;
+  onMouseMove: () => void;
```

Comment:
useless diff or align with FilterMenuValueItem.props ?

Reply:
The props was missing but used only in the template so typescript didn't see it